deno/payloads/v10/channel.ts

```diff
@@ -165,7 +165,7 @@ interface APIDMChannelBase<T extends ChannelType> extends APITextBasedChannel<T>
 
 export type APIDMChannel = APIDMChannelBase<ChannelType.DM>;
 
-export interface APIGroupDMChannel extends APIDMChannelBase<ChannelType.GroupDM> {
+export interface APIGroupDMChannel extends Omit<APIDMChannelBase<ChannelType.GroupDM>, 'name'> {
```

Comment:
Shouldn't need to omit 👀, is it actually necessary?

Reply:
Yep
![image](https://user-images.githubusercontent.com/42935195/155858598-e2d10602-b926-4c38-abd3-d384debc62ec.png)